test/verify/check-machines-dbus

```diff
@@ -791,6 +801,10 @@ class TestMachinesDBus(machineslib.TestMachines):
 
         m.execute("echo {0} > /tmp/disk.xml; virsh attach-device --config --file /tmp/disk.xml subVmTest1".format(diskXML))
 
+        # VM is not running, so we need to reload the page
```

Comment:
This is really needed here?

Reply:
Oh yeah it's needed. The previous test before this PR was buggy. And only reason it passed was because some old value was kept and it was same as new value (for new pool), so you didn't notice this value changed. 
But now when I added some more stuff before this code, this bug became visible.